boards/snps/nsim_riscv/nsim_riscv_rmx500_defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Synopsys, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_XIP=n
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_MP_MAX_NUM_CPUS=1
```

Comment:
Isn't it 1 by default when SMP=n? Probably we don't need it.

Reply:
Done